vllm/v1/engine/output_processor.py

```diff
@@ -201,18 +201,37 @@ def abort_requests(
         self,
         request_ids: Iterable[str],
     ) -> list[str]:
-        request_ids_to_abort = []
-        for request_id in request_ids:
-            req_state = self.request_states.pop(request_id, None)
+        request_ids_to_abort = self.flatten_req_to_abort(request_ids)
+        self.handle_abort_reqs(request_ids_to_abort)
+        return request_ids_to_abort
+
+    def flatten_req_to_abort(self, req_ids: Iterable[str]) -> list[str]:
```

Comment:
Can we call this something more descriptive? `get_parent_and_children_reqs`?

Reply:
It should probably also reflect the fact that the parent request is being removed.

Reply:
>  the fact that the parent request is being removed

Yes, Do you have any good suggestions? How about `try_pop_parent`?